
target_home_positions: [4.8, 2.9, 1.0, 4.2, 1.5, 1.25] #home position, same as move_robot.py from kinova_control pkg
target_time: 10.0 # arbitrary
